CMakeLists.txt

```diff
@@ -129,8 +129,12 @@ cuda_add_library(gdf SHARED
     #src/windowedops.cu
 )
 
-
-target_link_libraries(gdf arrow)
+if (ARROW_NUMERIC_VERSION EQUAL 900)
```

Comment:
Should this be greater than  or equal to?

Reply:
Actually, just equal to. Only arrow 0.9 had this issue.

Reply:
Ahh OK, great.